### DISCUSSION TITLE
Live streaming for all wcfm stores listed

### DISCUSSION THREAD

**Discussion by usamaahmed5656** [inspireui/support#5362] [New Feature/Ideas Requests]
please integrate a feautre where seller can  stream live inside the app on a day of the week and customer can watch them live  selling their products ,they are selling on very cheap prices to increase their sales and admin can also charge a little amount from sellers for going live on the app

**Comment by minhcasi** [comment]
Hi, this is a great feature, may I know i you are using any live stream cloud/services?

**Reply by usamaahmed5656** [reply]
no i am not firm to a cloud service you can use anyone you feel ease to integrate ..but please first understand the complete idea

from manager app the seller has a option to go live when he hit go live ,when any of the seller is live then on fluxstore multivendor app category screen(store list screen) the live text should be shown with the store name ..there should also be a widget of showing live ongoing streams on front page of the customers app and by mean showing streaming on customer's app homepage i am not wanting to show the video streaming because it can increase the loadtime of homepage but a widget that can filter from store list of which sellers are live rightnow and show the name of the store with banner image with live text on it

the third and most important thing is when a seller is live streaming and customer is watching them on their fluxstore multivendor screeen ...soo the streaming screen should have a side bar which is scrollable and is showing all on sale products or may be all products of that particular store ,, this feature will help customers to buy easily the things the seller is showing on live streaming

1 ) sellers going live from manager app
2 ) customers are able to see which sellers are live on fluxstore multivendor
3 ) sidebar scrollable products screen on streaming screen

hope you understand the idea and like it becasue this will get fluxstore to next level

**Comment by usamaahmed5656** [comment]
no i am not firm to a cloud service you can use anyone you feel ease to integrate ..but please first understand the complete idea 

from manager app the seller has a option to go live when he hit go live  ,when any of the seller is live then on fluxstore multivendor app category screen(store list screen) the live text should be shown  with the store name ..there should also be a widget of showing live ongoing streams on front page of the customers app and by mean showing streaming on customer's app homepage i am not wanting to show the video streaming  because it can increase the loadtime of homepage  but a widget that can filter from store list of which sellers are live rightnow and show the name of the store with banner image with  live text on it 

the third and most important thing is when a seller is live streaming and customer is watching them on their fluxstore multivendor screeen ...soo the streaming screen should have a side bar which is scrollable and is showing all on sale products or may be all products of that particular store ,, this feature will help customers to buy easily the things the seller is showing on live streaming 

1 ) sellers going live from manager app 
2 ) customers are able to see  which sellers are live on fluxstore multivendor 
3 ) sidebar scrollable  products screen on streaming screen


hope you understand the idea and like it becasue this will get fluxstore to next level

**Comment by usamaahmed5656** [comment]
any update on this ?    @minhcasi
